### PR TITLE
3600 - 2

### PR DESCRIPTION
**Pull request by sametsJR**
## Номер
3600

## Номер задания
2

## Ссылка на видео с демо
https://youtu.be/2eEc5-Rto3s

## Комментарии
Основной функционал бота работает.

### CODE REVIEW

File: 3600/2/commands/check_number.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    end
+  end
+
+  def register_from_message(*words)
+    respond_with :message, text: register_session(words)
+  end
+
+  private
+
+  def register_session(number = nil, *)
```
Comment (anatoliliotych) [inline 3600/2/commands/check_number.rb]: strange method signature

**Comment by sametsJR** [comment]
> where is env.sample file?

The .env file is hidden and stored on my disk.
.env file sample: TOKEN='my_private:token'

File: 3600/2/commands/check_number.rb (hunk 1 above)
Comment (sametsJR) [reply 3600/2/commands/check_number.rb]: Renamed method and added new signature for it

File: 3600/2/commands/check_number.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  private
+
+  def add_number(number)
+    redis = Redis.new
+    check_registaration(redis, number.first.to_i)
+  end
+
+  def check_registaration(redis, number)
+    if redis.get(number) || session.key?(:number)
```
Comment (anatoliliotych) [inline 3600/2/commands/check_number.rb]: you can avoid using if else if you make return blabla if

Comment (sametsJR) [reply 3600/2/commands/check_number.rb]: Smth like this?: return 'You have already registred' if redis.get(number) || sessionkey?(:number).
How many returns can be in one method?

Comment (anatoliliotych) [reply 3600/2/commands/check_number.rb]: as you wish, but it should be pretty clear for everyone.

Comment (sametsJR) [reply 3600/2/commands/check_number.rb]: I rewrote the method yesterday...

**Pull request merged**